### PR TITLE
require flake8 compliance

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: mvlearn/embed/splitae.py [hunk 1]
```diff
@@ -13,10 +13,15 @@
 # limitations under the License.
 
 import sys
+import itertools
+
+import numpy as np
+# XXX  I would use a nested import for matplotlib to make it a soft dep
```
[the remark this answers is not in the document]
Comment (rflperry) [reply mvlearn/embed/splitae.py:19]: i.e. moving the import line to `__init__` in a conditional or moving it to the conditional in `fit` where plotting occurs once? I don't know which is preferred but makes sense to me

**Comment by rflperry** [comment]
re-trigger travis by closing/opening

File: mvlearn/embed/splitae.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (agramfort) [reply mvlearn/embed/splitae.py:19]: just import `plt` in the function where you use `plt.foobar`

**Comment by agramfort** [comment]
@rflperry green now.

**Pull request merged**